rockpro64-dev.report

```diff
@@ -0,0 +1,9 @@
+BOOT=yes
+VERSION=5.59.180829
+KERNEL=4.4.152-rk3328
+NETWORK=yes
+WIRELESS=no
+HDMI=yes
+USB=yes
+DVFS=NT
```

Comment:
can easaly be tested (sudo armbianmonitor -m and a second shell with stress -c 4 (on and off)